airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/kotlin/io/airbyte/integrations/base/destination/typing_deduping/PostImportAction.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.base.destination.typing_deduping
+
+enum class PostImportAction {
```

Comment:
naming nit: InsertMode, or something along those lines? (append vs deduped)

post import action is tied closely to our implementation. In principle we could directly upsert records, etc